main.py

```diff
@@ -20,6 +20,25 @@ def one_week_from_now():
     next_week = datetime.today() + timedelta(days=7)
     return next_week.strftime(r"%Y-%m-%d")
 
+def get_next_week_days():
+    result = []
+    
+    if not config["days_of_week"]:
+        result.append(one_week_from_now())
+
+    for dow in config["days_of_week"]:
+        next_day = datetime.today()
+        
+        days_left = dow - next_day.weekday()
+        
+        if days_left <= 0:
+            days_left += 7
+            
+        next_day += timedelta(days=days_left)
+        
```

Comment:
This calculation is a bit all around. Can I suggest a denser form?

```suggestion
        today = datetime.today()
        days_left = (dow - today.weekday()) % 7
        next_day = today + datetime.timedelta(days=days_left)
```

Reply:
Of course.